client/app/web/content/styles/common.scss

```diff
@@ -0,0 +1,13 @@
+@import './reset.scss';
+
+body,
+html,
+body > div {
+    height: 100%;
+    max-height: 100%;
+}
+
+body {
+    font-family: Arial;
+    background: #242424;
+}
```

Comment:
Recalling i found a problem. With that structure we must explain CSS Modules and SASS in the presentation... We might be running out of time

Reply:
Do you prefer plain css?